README.dev.md

```diff
@@ -80,7 +80,10 @@ virtualenv -p python3 .venv
 .venv/bin/pip3 install -r requirements.txt
 ```
 ### Setup on Windows 10
-First make sure [Python](https://python.org) 3.7 or greater is installed (in the Python installer window be sure to click on "Add Python to PATH"), then open a PowerShell window and enter the following commands:
+First make sure [Python](https://python.org) 3.7 or greater is installed (in the Python installer window be sure to click on "Add Python to PATH").
+
+#### Python virtual environment
```

Comment:
This structure (Python vs Conda virtual env) would need to be duplicated in the Linux and Mac sections as well.